src/style.css

```diff
@@ -1,3 +1,56 @@
+.logo {
+  display: block;
+  padding: 10px 0;
+  height: 40px;
+  width: 40px;
+}
+
 body {
   margin: 0;
+  --hex-color: #00acdc;
+  font-family: "Times New Roman", Times, serif;
```

Comment:
why you add here this font, you don't use it anyway

Reply:
that was supposed to be fallback font, change it